app.js

```diff
@@ -19,6 +20,7 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
+app.use(helmet());
```

Comment:
Nice work adding in `helmet`. Question, is this mounted early enough? Should it be above the other app middleware?

Reply:
I was thinking that myself but I wasn't sure